config/config.go

```diff
@@ -165,6 +171,9 @@ func (c *Config) CheckMissingResourceEnvvars() {
 	if (c.Handler.Slack.Token == "") && (os.Getenv("SLACK_TOKEN") != "") {
 		c.Handler.Slack.Token = os.Getenv("SLACK_TOKEN")
 	}
+	if !c.Resource.Secret && os.Getenv("KW_SECRET") == "true" {
```

Comment:
nit: suggest moving this together with other `Resource`s, ie ~6 lines above.

Reply:
Makes sense.